### PR TITLE
Partially test consistency of grapheme cluster segmentation with canonical equivalence, and fix it for LGCs

### PR DESCRIPTION
**Pull request by eggrobin**
Towards #522.

This would have caught the issue reported by Charlotte Buff in https://www.unicode.org/L2/L2023/23159-pubrev.html#ID20230502072316, as well as unicode-org/properties#229.

In fact, it caught the corresponding issue for Tulu-Tigalari (which only affects _legacy_ grapheme clusters). See the failures in https://github.com/unicode-org/unicodetools/pull/645/commits/85eca4e4760cb9b369b992fc2cbdee10aa719a8d.

### CODE REVIEW

File: unicodetools/src/main/java/org/unicode/text/UCD/TestUnicodeInvariants.java [hunk 1]
```diff
@@ -46,6 +46,9 @@ public class TestUnicodeInvariants {
     private static int showRangeLimit = 20;
```
Comment (macchiati) [inline unicodetools/src/main/java/org/unicode/text/UCD/TestUnicodeInvariants.java:46]: Nice addition, and cleanup

**Review by macchiati — DISMISSED** [review]

File: unicodetools/src/main/java/org/unicode/text/UCD/TestUnicodeInvariants.java (hunk 1 above)
Comment (eggrobin) [reply unicodetools/src/main/java/org/unicode/text/UCD/TestUnicodeInvariants.java:46]: Note that I split that out into its own PR #646, which will thus need its own approval.

File: unicodetools/data/ucd/dev/PropList.txt [hunk 2]
```diff
@@ -1255,21 +1255,40 @@ FFFFE..FFFFF  ; Noncharacter_Code_Point # Cn   [2] <noncharacter-FFFFE>..<noncha
...
-1D16E..1D172  ; Other_Grapheme_Extend # Mc   [5] MUSICAL SYMBOL COMBINING FLAG-1..MUSICAL SYMBOL COMBINING FLAG-5
+1193D         ; Other_Grapheme_Extend # Mc       DIVES AKURU SIGN HALANTA
+11F41         ; Other_Grapheme_Extend # Mc       KAWI SIGN KILLER
+16FF0..16FF1  ; Other_Grapheme_Extend # Mc   [2] VIETNAMESE ALTERNATE READING MARK CA..VIETNAMESE ALTERNATE READING MARK NHAY
+1D165..1D166  ; Other_Grapheme_Extend # Mc   [2] MUSICAL SYMBOL COMBINING STEM..MUSICAL SYMBOL COMBINING SPRECHGESANG STEM
+1D16D..1D172  ; Other_Grapheme_Extend # Mc   [6] MUSICAL SYMBOL COMBINING AUGMENTATION DOT..MUSICAL SYMBOL COMBINING FLAG-5
 E0020..E007F  ; Other_Grapheme_Extend # Cf  [96] TAG SPACE..CANCEL TAG
 
-# Total code points: 127
+# Total code points: 152
```
Comment (eggrobin) [inline unicodetools/data/ucd/dev/PropList.txt:1291]: Note: 152-127=25; UTC decided on changing 18 characters, and there are 7 more Tulu-Tigalari characters new in 16.0.

**Review by markusicu — APPROVED** [review]
lgtm

**Pull request merged**